src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -374,18 +374,19 @@ populateProtoResponse(
                     if (txnMeta)
                     {
                         RPC::convert(*txnProto->mutable_meta(), txnMeta);
-
-                        auto amt = getDeliveredAmount(
-                            context,
-                            txn->getSTransaction(),
-                            *txnMeta,
-                            txn->getLedger());
-                        if (amt)
+                        if (!txnMeta->hasDeliveredAmount())
```

Comment:
Nit: the thing we _really_ care about is if `txnProto` has delivered amount. If that is easy to check, I'd rather check that.

Reply:
changed this